src/actions/setting.js

```diff
@@ -0,0 +1,7 @@
+export const SET_SETTING = "SET_SETTING"
+
+export const setSetting = (id, value) => ({
+	type: SET_SETTING,
+	id,
```

Comment:
Normalmente aquí tendrias 

```
{
  type: string,
  payload: { id, value }
}
```
Es recomendado cuando usas redux https://redux.js.org/style-guide/style-guide#reducers-should-own-the-state-shape básicamente porque luego puedes hacer el spread de las props del payload facilmente.

Echale un ojo también @lordpakixu